.vscode/tasks.json

```diff
@@ -38,33 +39,19 @@
         {
             "label": "clean",
             "type": "shell",
-            "command": "msbuild",
+            "command": "dotnet",
             "args": [
-                "/t:clean",
-            ],
-            "group": "build",
-            "presentation": {
-                "reveal": "silent"
-            },
-            "problemMatcher": "$msCompile"
-        },
-        {
-            "label": "force build linux",
-            "type": "shell",
-            "command": "msbuild",
-            "args": [
-                "/property:GenerateFullPaths=true",
-                "/p:Configuration=Debug",
-                "/p:DebugType=portable",
-                "/t:build",
-                "/p:ForceLinuxBuild=true"
+                "clean",
             ],
             "group": "build",
             "presentation": {
                 "reveal": "silent"
             },
             "problemMatcher": "$msCompile"
         },
+
+        // Below tasks are no longer being supported
+        // Succeeded by Lean CLI
```

Comment:
Was thinking we might just remove these all together?? Maybe even remove the scripts themselves since they are built to run lean with mono debugger not dotnet.

This also means we should address pycharm integration since it also makes use of these scripts.

Reply:
Could possibly adapt them, but I believe all the important functionality was implemented in Lean CLI